sematic/examples/bert_fine_tune_scam/README.md

```diff
@@ -0,0 +1,5 @@
+# This is an example README file
```

Comment:
Please write some content here with some basic documentation including references to the original pre-trained model on HuggingFace.

Reply:
Sure, will be adding documentation.